eng/dockerfile-templates/runtime/3.1/Dockerfile.nanoserver

```
@@ -17,7 +17,7 @@ RUN powershell -Command `
         }; `
         `
         mkdir dotnet; `
-        tar -C dotnet -oxzf dotnet.zip; `
+        tar -oxzf -C dotnet dotnet.zip; `
```

Ordering of the parameters is inconsistent here.  

```suggestion
        tar -oxzf dotnet.zip -C dotnet; `
```

---

Fixed